src/runtime/cpu/kernels/kernels.hpp

```diff
@@ -89,6 +89,23 @@ void CPUReLUForward(ConstVector X, Vector result);
 */
 void CPUReLUInputGrad(ConstVector input, ConstVector grad_output, Vector grad_input);
 
+/*
+ * CPUELUForward
+ * implements computation of ELU layer output
+ * X is a 1D vector [size]
+ * result have size [size]
+*/
+void CPUELUForward(ConstVector X, Vector result, float alpha);
```

Comment:
Do we need to add documentation for `alpha`?

Reply:
Done.